### ISSUE TITLE
How is this supposed to work?

### ISSUE THREAD

**Issue by KPSquall** [tewtal/alttp_sm_combo_randomizer#132]
Console or Emulator:snes9x

 If emulator state which emulator and version.

Original Filename:

Seed:

Item, Entrance or Custom: 

Mode: open
Logic: alttp-no-glitches-v10.2 sm-normal
Goal: ganon
Difficulty: normal
Variation: combo

Description of problem: cannot get morph ball without having morph ball?

Spoiler (if available): 

1 | Light World | Sahasrahla's Hut - Left | Super Missile
-- | -- | -- | --
1 | Light World | Blind's Hideout - Left | Lamp
1 | Light World | Blind's Hideout - Right | Morph Bombs
1 | Light World | Blind's Hideout - Far Left | Magic Cape
1 | Light World | Bottle Merchant | Progressive Sword
1 | Light World | Lost Woods Hideout | Bombos
1 | Light World | Floodgate Chest | Flippers
1 | Light World | Mini Moldorm Cave - Far Right | Flute
1 | Light World | Ice Rod Cave | Moon Pearl
1 | Hyrule Castle | Hyrule Castle - Map Chest | Varia Suit
2 | Light World | Waterfall Fairy - Right | Bow
2 | Norfair | Energy Tank (Hi-Jump Boots) | Power Bomb
3 | Castle Tower | Agahnim | Defeat Agahnim
4 | Dark Palace | Palace of Darkness - The Arena - Bridge | Palace of Darkness Big Key
5 | Dark Palace | Palace of Darkness - Dark Maze - Top | Hookshot
6 | Death Mountain | Paradox Cave Upper - Left | Progressive Glove
6 | Death Mountain | Paradox Cave Upper - Right | Cane Of Somaria
6 | Dark World | C-Shaped House | Screw Attack
6 | Dark World | Hype Cave - NPC | Pegasus Boots
7 | Crateria | Missile (Crateria bottom) | Progressive Glove
7 | Brinstar | Missile (pink Brinstar top) | Progressive Sword
7 | Brinstar | Missile (pink Brinstar bottom) | Bottle (Empty)
8 | Hyrule Castle | Hyrule Castle - Zelda's Cell | Gravity Suit
8 | Ice Palace | Ice Palace - Freezor Chest | Ice Palace Big Key
8 | Lower Norfair | Super Missile (Gold Torizo) | Morphing Ball

**Comment by jmontano86** [comment]
You'll need to use the warp at East Mire Shed to get to Lower Norfair. You should have the flute and both Gloves to get there.

**Comment by KPSquall** [comment]
Right, the second set of gloves appears to be in Crateria, the former mother brain spot which appears to require bombs or power bombs to open up the little area downward. I tried shooting it with super missiles, and regular missiles, there's not really the space to get a shinespark going if I were to get the speed booster. I have had about 5-6 runs like this where they put things in places like this that make no sense to me, I have looked at the logics and seen other people ask questions; I have looked at the maps that tell you every warp point between the games. The logic for the playthrough in the spoilers lately have a lot of these. I haven't played hard logic difficulty since I am not good at several glitches, so I am really confused.
